frontend/src/index.tsx

```diff
@@ -3,6 +3,8 @@ import ReactDOM from "react-dom";
 import "./index.scss";
 import App from "./App";
 import reportWebVitals from "./reportWebVitals";
+import "leaflet/dist/leaflet.css";
+import "leaflet/dist/leaflet.js";
```

Comment:
Try removing these lines (separately). Does it still work without either of them? My gut tells me that at least the `.js` is included in the react leaflet library you import elsewhere